todo_app/templates/index.html

```diff
@@ -13,10 +13,17 @@ <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items  %}
         <li> Id: {{item.id}} Title: {{ item.title}} Status: {{item.status}}</li>
-        <form  action="/deleteitem" method ='POST'>
-          <input type =text name= "id" value = "{{item.id}}"/>
-         
-          <input type= "submit" value = "Delete"/>
+        <form  action="/movecarddoing" method ='POST'>
+          <input type ="hidden" name= "id" value = "{{item.id}}"/>
+          <input type= "submit" value = "Move Card To Doing"/>
+        </form>
```

Comment:
You could wrap these "move card" forms in `{% if ... %}` `{% endif %}` so that they only appear if the card does not already have that status.